waterlog/src/Components/Tank.js

```diff
@@ -2,31 +2,31 @@ import React from 'react';
 
 const Tank = (props)=>{
   const tankIcon = ()=>{
-    if(props.tank.waterLevel < 50)
+    if(props.tank.percentage < 50)
       return "images/tank_0.png"
-    else if (props.tank.waterLevel >= 50 && props.tank.waterLevel < 75)
+    else if (props.tank.percentage >= 50 && props.tank.percentage < 75)
       return "images/tank_50.png"
-    else if (props.tank.waterLevel >= 75 && props.tank.waterLevel < 95)
+    else if (props.tank.percentage >= 75 && props.tank.percentage < 95)
       return "images/tank_75.png"
     else
     return "images/tank_100.png"
   }
 
   const pumpStatusIndicator = () => {
-    if(props.tank.pumpOn===true && props.tank.waterLevel > 95)
+    if(props.tank.pumpOn===true && props.tank.percentage ===0)
       return "turn pump off";
```

Comment:
In places it is sentence case and places like here it is not. PLease make this sentence case

Reply:
done i just remove that method it wasn't doing anything i forgot to take it off while i was tasting something